deploy/examples/catalogsourceconfig.cr.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: "marketplace.redhat.com/v1alpha1"
 kind: "CatalogSourceConfig"
 metadata:
   name: "example"
+  namespace: "default"
```

Comment:
The metadata you added specifies that the namespace is default but it looks like the namespace is set to global in the spec. Is this discrepancy intended?

Reply:
The namespace under metadata should be the namespace where the CSC is created, presumably the namespace the marketplace operator resides in. The targetNamespace under spec is the namespace that the CSC points to, which is where the corresponding CatalogSource should be created.

If `global` is the namespace where the marketplace operator is being deployed, then metadata.namespace should be set to `global`

Reply:
Might be clearer to put the namespace `marketplace` rather than `default` if that's where the marketplace operator is intended to be deployed.

Reply:
Yes `marketplace` would be the better option. I also think rather than making piecemeal changes like this, it would be better if this was tackled from a holistic stand point of deploying the marketplace operators with proper naming prefixes etc.

Reply:
And I would also suggest adding `namespace: marketplace` for `operatorsource.cr.yaml` as well just to be consistent.